bin/resolve-dependencies

```diff
@@ -64,7 +64,11 @@ const mains = ["unpkg", "jsdelivr", "browser", "main"];
   }
   {
     const package = await resolve("apache-arrow@4");
-    console.log(`export const arrow = dependency("${package.name}", "${package.version}", "${package.export}");`);
+    console.log(`export const arrow4 = dependency("${package.name}", "${package.version}", "${package.export}");`);
+  }
+  {
+    const package = await resolve("apache-arrow@9");
+    console.log(`export const arrow9 = dependency("${package.name}", "${package.version}", "+esm");`);
```

Comment:
Arrow 10 is out so you can update to that.

Reply:
Going to stick with Arrow 9 until duckdb-wasm upgrades…

https://github.com/duckdb/duckdb-wasm/blob/d779c7d5b6758a2abade699d1b564e1140a68de1/packages/duckdb-wasm/package.json#L26